tools/ARIAtools/extractProduct.py

```diff
@@ -645,7 +645,7 @@ def tropo_correction(full_product_dict, tropo_products, bbox_file, prods_TOTbbox
             if os.path.exists(os.path.join(outDir,'lookAngle',product_dict[2][i][0])):
                 lookfile=gdal.Open(os.path.join(outDir,'lookAngle',product_dict[2][i][0])).ReadAsArray()
             else:
-                lookfile=gdal.Open(os.path.join(outDir,'lookAngle',product_dict[2][i][0])).ReadAsArray()
+                lookfile=gdal.Open(os.path.join(outDir,'lookAngle',product_dict[2][0][0])).ReadAsArray()
```

Comment:
@sssangha this does not cover the case when the first product does not cover it completely. Should this not be in line with what we do for time-series?